segmentstore/server/src/main/java/io/pravega/segmentstore/server/writer/SegmentAggregator.java

```diff
@@ -757,34 +758,34 @@ private boolean canContinueFlushingExcess() {
     }
 
     /**
-     * Returns a FlushArgs which contains the data needing to be flushed to Storage.
+     * Returns a {@link BufferView} which contains the data needing to be flushed to Storage.
      *
-     * @return The aggregated object that can be used for flushing.
+     * @return A {@link BufferView} to flush.
```

Comment:
It should be noted this can return null.

Reply:
Fixed.